parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -119,6 +119,7 @@ PROXY_VARS="{{GetProxyVariables}}"
 ENABLE_TLS_BOOTSTRAPPING="{{EnableTLSBootstrapping}}"
 ENABLE_SECURE_TLS_BOOTSTRAPPING="{{EnableSecureTLSBootstrapping}}"
 CUSTOM_SECURE_TLS_BOOTSTRAP_AAD_SERVER_APP_ID="{{GetCustomSecureTLSBootstrapAADServerAppID}}"
+ENABLE_KUBELET_SERVING_CERTIFICATE_ROTATION="{{EnableKubeletServingCertificateRotation}}"
```

Comment:
@tallaxes @Bryce-Soghigian FYI, additive change to cse_cmd.sh

Reply:
oops you already tagged Bryce earlier, I missed that!

Reply:
ty mr anuj

Reply:
It looks like if we don't include this flag at all, the current functionality is preserved (i.e. it is equivalent to setting it to `false`, right?